backend/models/bible_plan.py

```
@@ -327,3 +327,39 @@ async def get_published_reading_plans() -> List[ReadingPlanOut]:
     except Exception as e:
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=f"Error fetching published reading plans: {e}")
 
+
+async def create_template_from_plan(plan_id: str, user_id: str) -> Optional[ReadingPlanTemplateOut]:
+    """Create a template from an existing Bible plan"""
+    try:
+        # Get the plan
+        plan = await get_reading_plan_by_id(plan_id, user_id)
+        if not plan:
+            raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Plan not found")
```

Ruff (0.14.5)

Abstract `raise` to an inner function

(TRY301)